automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/monitor/SubmissionDetailsPage.scala

```diff
@@ -76,11 +76,11 @@ class SubmissionDetailsPage(namespace: String, name: String, var submissionId: S
   /**
     * Wait for Submission to complete. 15 seconds polling.
     *
-    * @param timeOut: Time out. Default set 25.minutes
+    * @param timeOut: Time out. Default set 35.minutes
```

Comment:
increase wait time to 35 minutes for a submission to finish with any submission status. timeout exception fails test. a failed test from timeout is not a real app defect.